include/Map.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "Tile.h"
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+using tileStdMap = std::map<std::pair,tile::Tile>;
```

Comment:
It is not necessary to include the namespace in the name of a variable
```suggestion
using tileMap = std::map<std::pair,tile::Tile>;
```